src/cli/utils.js

```diff
@@ -49,3 +49,15 @@ exports.getIPFS = (callback) => {
 exports.getRepoPath = () => {
   return process.env.IPFS_PATH || os.homedir() + '/.ipfs'
 }
+
+exports.createLogger = (visible) => {
+  return (msg, newline = true) => {
```

Comment:
Shouldn't the tests run against the compiled version that we distribute rather than the source we develop with? Anyways, I'll fix this but worth thinking about.